lib/hyacinth/storage/abstract_storage.rb

```diff
@@ -63,7 +64,11 @@ def delete(location)
       # Returns the first compatible storage adapter for the given location, or nil if no compatible storage adapter is found.
       def storage_adapter_for_location(location)
         adapter = @storage_adapters.find { |ad| ad.handles?(location) }
-        Rails.logger.error("Tried to find storage adapter for #{location}, but no adapter was found.")
+        if adapter.nil?
```

Comment:
```suggestion
        if adapter = @storage_adapters.find { |ad| ad.handles?(location) }
```

Reply:
probably `unless`?

Reply:
yup, totally right, i'll fix it in the suggestion

Reply:
Method updated as per our gchat conversation.  Thanks!